### ISSUE TITLE
fnlog couldn't print slice of struct

### ISSUE THREAD

**Issue by annopkomol** [finnomena/go-fnlog#9]
```
package main

import "github.com/finnomena/go-fnlog"

func main() {
	type a struct {
		a int
	}
	b := []a{a{}, a{}}
	fnlog.Debug(b)
}

```
when i try to print the following code, it gives me this
`panic: runtime error: hash of unhashable type []main.a`

**Comment by tongium** [comment]
What version did you use?

**Comment by annopkomol** [comment]
v 0.1.0

**Comment by thedevper** [comment]
#10 Append map read, write lock issue

**Comment by annopkomol** [comment]
fixed on #10
